code/modules/atmospherics/machinery/atmospherics.dm

```diff
@@ -189,28 +189,34 @@ Pipelines + Other Objects -> Pipe network
 		var/I = int_air ? int_air.return_pressure() : 0
 		var/E = env_air ? env_air.return_pressure() : 0
 		var/internal_pressure = I - E
+		var/pipeslowdown = 1
 
 		playsound(src.loc, W.usesound, 50, 1)
 		to_chat(user, "<span class='notice'>You begin to unfasten \the [src]...</span>")
+
+		for(var/obj/item/clothing/shoes/magboots/usermagboots in user.get_equipped_items())
+			if(usermagboots.gustprotection && usermagboots.magpulse)
+				safefromgusts = TRUE
+
 		if(internal_pressure > 2*ONE_ATMOSPHERE)
-			to_chat(user, "<span class='warning'>As you begin unwrenching \the [src] a gust of air blows in your face... maybe you should reconsider?</span>")
 			unsafe_wrenching = TRUE //Oh dear oh dear
+			if(internal_pressure > 1750 && !safefromgusts) // 1750 is the pressure limit to do 60 damage when thrown
+				to_chat(user, "<span class='userdanger'>As you struggle to unwrench \the [src] a huge gust of gas blows in your face! This seems like a terrible idea!</span>")
+				pipeslowdown = 3 // give them more time to decide this is a bad idea, since it might kill them
+			else
+				to_chat(user, "<span class='warning'>As you begin unwrenching \the [src] a gust of air blows in your face... maybe you should reconsider?</span>")
 
-		if(do_after(user, 40 * W.toolspeed, target = src) && !QDELETED(src))
+		if(do_after(user, 40 * W.toolspeed * pipeslowdown, target = src) && !QDELETED(src))
```

Comment:
not sure how I feel about this new delay. This will take 12 full seconds in order to unwrench a pipe which is a pretty long time. Consider bumping it down to `2` (8 seconds) or `2.5` (10 seconds)

Reply:
This only happens if you're not wearing magboots and theres extreme pressure. Wearing safe magboots would result in only 4 seconds.

Reply:
I know, I'm just saying that 12 seconds is quite a long time even for the extreme pressure

Reply:
I get it's a warning, but I would not x3 the time. I would keep it as is.